### DISCUSSION TITLE
Wyszukiwanie zamówienia po numerze przesyłki

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#5466] [Suggestions - rejected]
Hej!
Mamy bardzo dużo przypadków (kilkadziesiąt dziennie), że klienci nie odbierają przesyłki.
Chcielibyśmy odnaleźć zamówienie po numerze przesyłki, żeby szybko zwrócić wpłatę. Niestety nigdzie w API nie widzę takiej możliwości.
Czy możecie taką opcję dodać?

Pozdrawiam!

**Comment by jakubmaguza** [comment]
@Lukasz-Zurek Możesz się do tego odnieść?

**Reply by Lukasz-Zurek** [reply]
Mogę potwierdzić, że nie można obecnie odnaleźć zamówienia po numerze przesyłki. Temat jest nam znany. Przekazałem go już do odpowiedniego zespołu. Mam nadzieję, że uda się taką opcję dodać.

**Comment by MartaNowaczyk** [comment]
Nie planujemy wdrożyć takiej zmiany. Mamy za to odwrotne działanie czyli [zwracanie numerów przesyłek dodanych do zamówienia](https://developer.allegro.pl/documentation#operation/getOrderShipmentsUsingGET).
